### PR TITLE
chore: add a link to widget typeform

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Just put https://cowprotocol.typeform.com/to/rONXaxHV everywhere instead of `TODO-TYPEFORM`

### CODE REVIEW

**Review by anxolin — COMMENTED** [review]
Cool!

should we add it here too? 
image

**Comment by shoom3301** [comment]
image

@anxolin great suggestion! Added it

File: apps/widget-configurator/src/app/embedDialog/utils/sanitizeParameters.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { CowSwapWidgetParams } from '@cowprotocol/widget-lib'
+
+export function sanitizeParameters(params: CowSwapWidgetParams) {
+  return {
+    ...params,
+    provider: `<eip-1193 provider>`,
+    '//': 'Please contact https://cowprotocol.typeform.com/to/rONXaxHV to enable your partner fee.',
```
Comment (anxolin) [inline apps/widget-configurator/src/app/embedDialog/utils/sanitizeParameters.ts:7]: Isn't there any other way to add the comment? the code is JS

Comment (shoom3301) [reply apps/widget-configurator/src/app/embedDialog/utils/sanitizeParameters.ts]: image

Fixed

Comment (anxolin) [reply apps/widget-configurator/src/app/embedDialog/utils/sanitizeParameters.ts]: I made a PR on top. If you agree with that one, we can merge this 2 PRs: https://github.com/cowprotocol/cowswap/pull/3345

**Review by anxolin — APPROVED** [review]

**Pull request merged**